### DISCUSSION TITLE
It would be nice if it could work if you own multible monitors

### DISCUSSION THREAD

**Discussion by RomeoAlphaEkko** [Kyrluckechuck/TFT-Bot#27] [Ideas]
Hi,

if you own multible monitors, like 2 and run it on your main monitor,

it does not work unless you disable your second monitor.

It would be cool if one could avoid this.

tx in advance (if you have any ideas, else still tx)

**Comment by Kyrluckechuck** [comment]
Weirdly I actually have this working on my left-most monitor and regularly have both monitors on. That said, I know there are some chunks of this that are coded specifically for coordinates on the screen, and not the individual screen the executable is on (relative coordinates) -- definitely something I want to look into!

**_Edit:_** This has since been implemented via https://github.com/Kyrluckechuck/TFT-Bot/pull/140, and has been included in the main branch for a while now 😃

**Comment by RIPbyCode** [comment]
technically it should be possible to use this script on Bluestacks 
There you can use multiple instances

**Reply by Kyrluckechuck** [reply]
I'm not certain they're looking to run multiple instances, I think this is specifically referring to the bot not being able to work properly on some multi-monitor setups depending on the "order" windows has the monitors set up in. 

As an aside, this discussion point is actually moot as this discussion was resolved / feature was likely resolved by https://github.com/Kyrluckechuck/TFT-Bot/pull/140! 😃
